.gitignore

```diff
@@ -1,3 +1,4 @@
 dist
 node_modules
+bower_components
```

Comment:
if you don't prepend it with a /, then it recursively ignores the file names. So it'll catch common/bower_components